### DISCUSSION TITLE
Idee: Zeitsteuerung für Power Limit/ Akku für Nachtgrundbedarf

### DISCUSSION THREAD

**Discussion by Narcodoc** [hoylabs/OpenDTU-OnBattery#694] [Ideas]
Meine Idee wäre eine Zeitsteuerung des Power Limits. Tagsüber Power Limit über Dynamic Limiter und Nachts wenn der Akku läuft  Grundbedarf: z.B. 150Watt ab Sonnenuntergang. Müsste gut zu realisieren sein.

**Comment by spcqike** [comment]
Welchen Vorteil soll das bringen? 
Warum nicht nachts auch einfach den DPL aktiv haben?

**Comment by redvari1** [comment]
Ich find die Idee auch super, nur genau umgekehrt.

Mein Szenario: Ich habe einen Akku, der nachts nichts in das Netz einspeisen soll, daher auch die Limitierung des WR. Tagsüber wär mir eine Einspeisung egal, daher kann der WR gern mit 100% laufen.
